build.proj

```diff
@@ -16,10 +16,10 @@
     /t:Publish
       Creates local nuget packages and MSI
 
-    /p:CodeSign=True
+    /p:CodeSignESRP=True
```

Comment:
Let's levae thsi switch as CodeSign - this will minimize the need for updating documentation and calling scripts / Jenkins/ADO tasks